app/presenters/project_presenter.rb

```diff
@@ -0,0 +1,7 @@
+class ProjectPresenter < SimpleDelegator
```

Comment:
I don't think this is, but we are collecting helpers that have to handle when values are nil and I think this pattern helps consolidate those helper concepts in to this one class, I would like to see us come back and add the two methods from `ProjectInformationHelper` here.

With hindsight, this might be better named `ProjectInformationPresenter`.

Not super wedded to this though, it's kind of swings and roundabouts.

https://www.rubyguides.com/2019/09/rails-patterns-presenter-service/